app/views/hosts/_provisioning.html.erb

```diff
@@ -4,7 +4,7 @@
   <h4><%= _('Templates resolved for this operating system') %></h4>
   <% templates.sort{|t,x| t.template_kind <=> x.template_kind}.each do |tmplt| %>
       <div class="form-group">
-        <div class="control-label col-md-8"> <%= _("%s Template") % tmplt.template_kind %>
+        <div class="control-label col-md-8"> <%= _("%s") % tmplt.template_kind %>
```

Comment:
When the string is only an interpolation like here, you don't need the interpolation, you could use the variable directly:
```suggestion
        <div class="control-label col-md-8"> <%= _(tmplt.template_kind) %>
```